src/fedora/34/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-34
+
+# Install Helix Dependencies
+
+RUN dnf install -y \
+        python3 \
+        python3-devel \
+        libatomic \
+        libffi-devel \
+        redhat-rpm-config \
+        sudo && \
+    dnf clean all
+
+RUN ln -sf /usr/bin/python3 /usr/bin/python && \
+    curl https://bootstrap.pypa.io/get-pip.py -o ./get-pip.py --fail --silent --show-error && \
+    python ./get-pip.py && rm ./get-pip.py && \
+    python -m pip install --upgrade pip==20.2 && \
+    python -m pip install virtualenv==16.6.0 && \
+    pip download --no-deps helix-scripts --index-url https://dnceng.pkgs.visualstudio.com/public/_packaging/helix-client-prod/pypi/simple && \
+    pip install ./helix_scripts-*-py3-none-any.whl
+
+# This is temporary until we have flow to packages.microsoft.com
+
+RUN dnf install -y dotnet gem lttng-tools perl-FindBin rpmdevtools ruby-devel && \
```

Comment:
FYI: this pulls down the Fedora build of .NET. Since you are adding the Microsoft packages repo to this container, I am not sure if it's intentional.

Reply:
That is ok. msquic only need global dotnet executable to install and run `clog`. I'm hoping this section will go away in few weeks (e.g. before 6.0 ships)

Reply:
Okay. Now that I think about it, what it pulls down might be non-deterministic. If it pulls down a mix of packages (some from Fedora, others from Microsoft) that might not work too well. You might want to verify that the SDK that gets installed into this container is working. See https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-mixup for more details/fixes.

Reply:
seems to be fine. The dotnet is removed immediately after build on line 32. So it should not interfere with Helix or anything else. Using the OS package seems most natural to me. I would do it for PowerShell as well but I could not find anything direclty from Fedora.